website/src/documentation/patterns/side-input-patterns.md

```diff
@@ -22,11 +22,11 @@ limitations under the License.
 
 The samples on this page show you common Beam side input patterns. A side input is an additional input that your `DoFn` can access each time it processes an element in the input `PCollection`. For more information, see the [programming guide section on side inputs]({{ site.baseurl }}/documentation/programming-guide/#side-inputs).
 
-## Using global window side inputs in non-global windows
+## Slowly updating global window side inputs
```

Comment:
Changing this header here broke the link here: https://github.com/apache/beam/blame/39d695f040227a16766b83a36bf05c5f6d688000/website/src/documentation/patterns/overview.md#L30

You'll need to fix link to match your change to the header or revert your change to the header.